### DISCUSSION TITLE
High cardinality features

### DISCUSSION THREAD

**Discussion by TashowarTanzim** [pycaret/pycaret#3527] [Q&A]
In the previous version of pycaret (2.3.10), I was able to specify some of my highly cardinal features (i.e. categorical features with a lot of levels) within the setup() function with the parameter :"high_cardinality_features = [....]". However, I get an argument error with this parameter while I try to use it in an environment with the latest version of pycaret (version 3.0). Any idea what has changed, and how exactly can I prevent one-hot encoding of highly cardinal features. I could not find anything in the documentation of Pycaret 3.0

**Comment by ngupta23** [comment]
Let's track here: https://github.com/pycaret/pycaret/issues/3528
